nettacker/database/db.py

```
@@ -70,17 +96,35 @@ def send_submit_query(session):
-                return True
-            except Exception:
-                time.sleep(0.1)
-    except Exception:
-        log.warn(messages("database_connect_fail"))
+    if isinstance(session, tuple):
+        connection, _ = session
+        try:
+            for _ in range(1, Config.settings.max_submit_query_retry):
+                try:
+                    connection.execute("COMMIT")
+                    return True
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)